src/app/page/MakeMission/MissionForm.js

```diff
@@ -88,15 +92,15 @@ function MissionForm({ handleChange, values, onSubmit, getFile /*, assignHelper,
       return;
     }
     if (stage === "pickUp") {
-      setPickUpDateLabel(date);
+      setPickUpDateLabel(date.toString().substr(0, 15));
```

Comment:
According to lines `97` and `107` looks like `data` could be undefined so may want to have a fallback on these sets as well

Reply:
Good catch - Resolved this by adding a condition that checks if `date` is undefined before assigning values